source/renderer/app/components/staking/delegation-count-down/DelegationCountDownInfo.js

```diff
@@ -5,48 +5,82 @@ import { defineMessages, intlShape } from 'react-intl';
 import { Button } from 'react-polymorph/lib/components/Button';
 import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
 import humanizeDuration from 'humanize-duration';
-import moment from 'moment';
 import styles from './DelegationCountDownInfo.scss';
+import { ROUTES } from '../../../routes-config';
 
 const messages = defineMessages({
   heading: {
-    id: 'paper.delegation.notification.heading',
+    id: 'staking.delegationCountDown.heading',
     defaultMessage: '!!!Cardano decentralization',
     description: 'Headline for the Decentralisation notification.',
   },
-  info: {
-    id: 'paper.delegation.notification.info',
+  description: {
+    id: 'staking.delegationCountDown.description',
     defaultMessage:
       '!!!Cardano will soon start its transition from a federated to a decentralized system. This will mark the start of stakeholders being able to earn rewards for participating in the running of the network. They will be able to participate directly in the process of staking or can delegate their stake to stake pools to earn rewards in ada.',
     description: 'Info for the Decentralisation notification.',
   },
   timeLeftDesc: {
-    id: 'paper.delegation.notification.timeLeftDesc',
+    id: 'staking.delegationCountDown.timeLeftDesc',
     defaultMessage: '!!!Rewards begin in',
     description: 'Description for the Decentralisation notification.',
   },
   buttonLabel: {
-    id: 'paper.delegation.notification.buttonLabel',
+    id: 'staking.delegationCountDown.buttonLabel',
     defaultMessage: '!!!Learn more',
     description: 'Button Label for the Decentralisation notification.',
   },
 });
 
-type Props = { currentLocale: string, startDateTime: string };
+type Props = { actions: any, currentLocale: string, startDateTime: string };
 
 @observer
 export default class DelegationCountDownInfo extends Component<Props> {
   static contextTypes = {
     intl: intlShape.isRequired,
   };
 
-  translateTimeLeft = () => {
-    const { currentLocale, startDateTime } = this.props;
+  timeoutHandler = null;
+  state = { timeLeft: null };
+
+  componentDidMount() {
+    this.updateTimeLeft();
+  }
+
+  updateTimeLeft = () => {
+    const { actions, startDateTime } = this.props;
     const timeLeft = Math.max(
       0,
-      moment(startDateTime).valueOf() - new Date().getTime()
+      new Date(startDateTime).getTime() - new Date().getTime()
     );
 
+    this.setState({ timeLeft });
+
+    if (timeLeft === 0) {
```

Comment:
Besides sending only one action, you can actually create a new Action for this part of the code. You would create a **StakingActions** file and add an action like `goToStakingInfo`. Inside **StakingStore** you'd listen to that action and use the `goToRoute` action.